module/registry.mjs

```diff
@@ -61,11 +61,72 @@ class EnchantmentRegisty {
   }
 }
 
+/* -------------------------------------------- */
+/*  Message Rolls                               */
+/* -------------------------------------------- */
+
+class RollMessageRegistry {
```

Comment:
I have a preference for calling this just `MessageRegistry` and exporting it as `registry.messages`. It's terser, and maybe we want to repurpose it later. I know it currently only works with roll messages due to its reliance on the `roll.type` flag.